packages/col/src/col.ts

```diff
@@ -84,7 +84,7 @@ const ElCol = defineComponent({
         }
       })
       // this is for the fix
-      if (gutter) {
+      if (gutter.value) {
```

Comment:
What is the purpose of this code?
I don't think this code is useful and can be removed